board/rockchip/evb_rk3576/evb_rk3576.c

```diff
@@ -103,7 +103,8 @@ int board_usb_cleanup(int index, enum usb_init_type init)
 
 #ifdef CONFIG_ID_EEPROM
 static struct variant_def variants[] = {
-	{"radxa,cm4-io", 0, 40, 0, -1, "rockchip/rk3576-radxa-cm4-io.dtb"},
+	{"radxa,cm4-io", 0, 40, 2000, 2080, "rockchip/rk3576-radxa-cm4-io.dtb"},
+	{"radxa,rpi-cm4-io", 0, 40, 4045, 4097, "rockchip/rk3576-radxa-cm4-rpi-cm4-io.dtb"},
```

Comment:
```suggestion
	{"radxa,cm4-io", 0, 40, 4045, 4097, "rockchip/rk3576-radxa-cm4-rpi-cm4-io.dtb"},
```
之前的设置应该是无效的
另外树莓派上这个adc是怎么接的，能确保读这么多么

Reply:
@chendadap compatible是这个  radxa,cm4-rpi-cm4-io  
验证是要确保启动加载的是对应的dtb
通过cat /proc/device-tree/model    确认

Reply:
已修改,已确认@Ken-Vamrs